src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "ngraph/runtime/intelgpu/intelgpu_backend.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+extern "C" void create_backend()
```

Comment:
`create_backend` should be in a C header included in an `extern "C"` by `intelgpu_backend.hpp`. Then you can leave it out here, and you don't need to worry about headers and implementations getting out of sync.

You also want it to be `create_backend(void)` since in C `create_backend()` means you can pass anything you want to the function.

Reply:
I did this in the way the same as for all other backends. If you like refactor the code I could do this in separate PR.